tests/v2/e2e/assets/unary_crud.yaml

```
@@ -0,0 +1,369 @@
+      - name: Inset Operation
+        executions:
+          - name: Insert
+            type: insert
+            mode: unary
+            parallelism: 10
+            num: 30000
+            wait: 2m
+          - mode: unary
+            name: IndexInfo
+            type: index_info
+        name: Insert -> IndexInfo
```

YAMLlint (1.35.1)

[error] duplication of key "name" in mapping

(key-duplicates)